callisto/delivery/api.py

```diff
@@ -20,33 +43,25 @@ class DeliveryApi(object):
         DeliveryApi maps to
     '''
 
-    def __init__(self):
-        self.set_api_class()
+    api_env_variable = 'CALLISTO_NOTIFICATION_API'
+    default_classpath = 'callisto.delivery.api.AbstractNotification'
```

Comment:
I flip back and forth on this, but given the large amount of other setup implementing callisto-core will require, I'm leaning towards "no" on this. This is one of the things implementers should be very aware of needing to customize